Core/clim-core/commands.lisp

```diff
@@ -847,16 +847,15 @@ examine the type of the command menu item to see if it is
                (nil)
              (destructuring-bind (,command-name ,@original-args)
                  ,partial-command
-               (let ((,command-line-name (command-line-name-for-command
-                                          ,command-name
-                                          ,command-table
-                                          :errorp nil))
-                     ,@(mapcar #'list required-arg-names original-args))
-                 (accepting-values (,stream :select-first-query t
-                                            :align-prompts t)
-                   (format ,stream
-                           "You are being prompted for arguments to ~S~%"
-                           ,command-line-name)
+               (let* ((,command-line-name (command-line-name-for-command
+                                           ,command-name
+                                           ,command-table
+                                           :errorp nil))
+                      (,label (format nil
+                                      "You are being prompted for arguments to ~S"
```

Comment:
Since table starts exactly where the cursor is, You may want to add a new line format directive here, Otherwise the result will look odd (To See it click on: german town example -> Get Distance)

Reply:
that's what I meant by terpri in the other place
```
(when label
                        (format stream label)
                        (terpri))
```
but forgot to pass `stream` to terpri (fixed in last commit).

Reply:
That's the second reason I hate terpri. (The first--- it got it's own homepage at http://terpri.org )